itest/list_on_test.go

```diff
@@ -411,6 +415,10 @@ var allTestCases = []*lntest.TestCase{
 		Name:     "send payment keysend mpp fail",
 		TestFunc: testSendPaymentKeysendMPPFail,
 	},
+	{
+		Name:     "send payment routehints keysend",
```

Comment:
also this one, we don't need this, you change focused on the `lncli` only, no need to test the already working `lnd` side. Can be removed.